src/common/PkmImage/PkmImage.tsx

```diff
@@ -32,18 +32,16 @@ const PkmImage = ({ dexNum, form, name, shiny = false }: PkmImageProps) => {
   };
 
   useEffect(() => {
-    loadImage(imgSrc)
+    loadImage(baseImgSrc + getImage(dexNum, form))
+      .then((src) => {
+        setImgSrc(src as string);
+      })
       .catch((error) => {
         console.error(error);
```

Comment:
I like to bubble up my errors in Promises. Not a recommendation for this review, but something that will be fun to look into.